src/step.js

```diff
@@ -361,8 +361,8 @@ wgxpath.Step.Axis = {
         var testName = test.getName();
         // IE8 doesn't allow access to the style attribute using getNamedItem.
         // It returns an object with nodeValue = null.
```

Comment:
I assume this is also IE8 specific, so can you extend this comment to say that .style can mutate the DOM?